lib/capabilities.go

```diff
@@ -78,3 +91,31 @@ func CapAwayNotify(caps *CapManager) {
 		},
 	)
 }
+
+func CapServerTime(caps *CapManager) {
+	name := "server-time"
+	caps.Supported[name] = ""
+
+	caps.FnsInitListener[name] = func(listener *Listener) {
+		listener.TagsEnabled = true
+	}
+
+	caps.FnsMessageToClient = append(
+		caps.FnsMessageToClient,
+		func(listener *Listener, message *ircmsg.IrcMessage) bool {
+			if !listener.IsCapEnabled(name) {
+				return false
+			}
+
+			_, exists := message.Tags["time"]
+			if !exists {
+				message.Tags["time"] = ircmsg.TagValue{
```

Comment:
Is there an actual difference between `name=` and `name`? As far as I can see they both mean exactly the same - a tag without a value.